### ISSUE TITLE
Unusable

### ISSUE THREAD

**Issue by sanbotto** [rendermani/ascio-ssl-whmcs-plugin#1]
Hi! I installed this on my WHMCS but I don't know where to start debugging since I get no error messages.

I added the missing options of asciossl_ConfigOptions but that was not enough. I also realized that the main directory name needed to be "asciossl". Now, the only thing I have left is to check the whole code for other missing parts of the code since, as already stated, the plugin outputs no error.

Could you please check it out?

Thanks!

PS: I'm running Apache 2.4 with PHP 7.2 and MySQL 5.7, but the errors do not appear to be related to incompatibility rather than missing code.

**Comment by sanbotto** [comment]
After a lot of digging, I realized that adding the missing config fields breaks the module.

Also, from the client area, I saw and error reporting that the table **mod_asciossl_settings** did not existed. That lead me to search for "mod_asciossl_settings" on the files of the module and I noticed that install.sql was not passed to MySQL by install.php (neither there was an indication to do it manually). Manually running the SQL via phpMyAdmin fixed all issues.

I'm now able to use the plugin without the Autoinstaller SSL, but I get invalid password error when submitting my SSL order (I can't correctly set the password due to the issue with the config options).
